src/ios/FileUploader.m

```diff
@@ -125,12 +124,11 @@ -(void)writeMultipartDataToTempFile: (NSURL*)tempFilePath
                             headers:(NSDictionary*)headers
                          parameters:(NSDictionary*)parameters
                             fileKey:(NSString*)fileKey
-                      requestMethod:(NSString*)requestMethod
                   completionHandler:(void (^)(NSError* error, NSMutableURLRequest* request))handler{
     AFHTTPRequestSerializer *serializer = [AFHTTPRequestSerializer serializer];
     NSError *error;
     NSMutableURLRequest *request =
-    [serializer multipartFormRequestWithMethod:requestMethod
+    [serializer multipartFormRequestWithMethod:@"POST"
```

Comment:
@zafirskthelifehacker is it something we want here ? Why did we change the iOS part ?

Reply:
@ombr this is a change from the version 3 (master) that will not be used. The v2 version had the method harcoded to  POST. This is why it is appearing as a diff, the master version is not the our Single source of truth but the branch v2 is .(our shadow master)

Reply:
@ombr we can change the default branch to v2 on the repo